src/Compiler/Compiler/Latte2Compiler.php

```diff
@@ -109,6 +114,25 @@ private function getEngineFiltersByReflection(): array
         return $engineFilters;
     }
 
+    /**
+     * @return array<string, callable>
+     * @throws ReflectionException
+     */
+    private function getEngineFunctionsByReflection(): array
```

Comment:
This magix is required because Engine holds list of functions in private property. It is same approach we already use for filters